apps/bundle-size/src/commands/build.js

```diff
@@ -0,0 +1,67 @@
+// @ts-check
+
+const chalk = require('chalk');
+const Table = require('cli-table3');
+const fs = require('fs-extra');
+const glob = require('glob');
+const path = require('path');
+const prettyBytes = require('pretty-bytes');
+
+const buildFixture = require('../utils/buildFixture');
+const { hrToSeconds } = require('../utils/helpers');
+const prepareFixture = require('../utils/prepareFixture');
+
+/**
+ * @param {{ verbose: boolean }} options
+ */
+async function build(options) {
+  const { verbose } = options;
+
+  const startTime = process.hrtime();
+  const artifactsDir = path.resolve(process.cwd(), 'dist', 'bundle-size');
+
+  await fs.remove(artifactsDir);
+
+  if (verbose) {
+    console.log(`${chalk.blue('[i]')} artifacts dir is cleared`);
+  }
+
+  const fixtures = glob.sync('bundle-size/*.fixture.js', {
+    cwd: process.cwd(),
+  });
+
+  if (verbose) {
```

Comment:
**thought for future:** rather than imperative checking , what should or should not be printed, we should come up with standardised approach how to log things out for node environments. debug package could help us a lot for example.

Reply:
Agree and support 👍